web/html/src/manager/maintenance/maintenance-windows-details.js

```diff
@@ -0,0 +1,178 @@
+/* eslint-disable */
+'use strict';
+
+const React = require("react");
+const {Button} = require("components/buttons");
+const {ModalButton} = require("components/dialog/ModalButton");
+const {DeleteDialog} = require("components/dialog/DeleteDialog");
+const { TopPanel } = require('components/panels/TopPanel');
+const {Messages} = require("components/messages");
+const {Form} = require("components/input/Form");
+const {Check} = require("components/input/Check");
+const { BootstrapPanel } = require('components/panels/BootstrapPanel');
+
+const scheduleTypeToString = (scheduleType) => {
+    switch (scheduleType) {
+        case "MULTI":
+            return "Multi";
+        case "SINGLE":
+            return "Single";
+    }
+    return null;
+}
+
+class MaintenanceWindowsDetails extends React.Component {
+
+    constructor(props) {
+        super(props);
+
+        this.state = {
+            type: props.type,
+            calendarData: this.props.data.calendarData,
+            messages: [],
+        };
+    }
+
+    setCheck = (model) => {
+        model.strategy === "" && (model.strategy = false);
+        this.setState(model);
+    };
+
+    addStrategy = () => {
+        const item = this.props.data;
+        item.strategy = (this.state.strategy ? "Cancel" : "Fail");
+        return item;
+    };
+
+    renderScheduleDetails(data) {
+        return (
+            <BootstrapPanel title={t("Schedule Details")}>
+                <div className="table-responsive">
+                    <table className="table">
+                        <tbody>
+                        <tr>
+                            <td>{t("Schedule Name")}</td>
+                            <td>{t(data.scheduleName)}</td>
+                        </tr>
+                        <tr>
+                            <td>{t("Assigned Calendar")}:</td>
+                            <td>{t(data.calendarName)}</td>
+                        </tr>
+                        <tr>
+                            <td>{t("Schedule Type")}:</td>
+                            <td>{t(scheduleTypeToString(data.scheduleType))}</td>
+                        </tr>
+                        </tbody>
+                    </table>
+                </div>
+                <DeleteDialog id="delete-modal"
+                              title={t("Delete maintenance schedule")}
+                              content={t("Are you sure you want to delete the selected item? \n" +
+                                  "Deleting the schedule will unassign all systems from this schedule.")}
```

Comment:
It works nice, but I would rephrase just to be less redundant:
`This will remove the current schedule from all the systems assigned to it.`